### ISSUE TITLE
Cronjob always tries to contact LDAP server

### ISSUE THREAD

**Issue by RalfJung** [opatut/dudel#94]
When I access `api/cron`, I get
```
error updating LDAP users: {'desc': "Can't contact LDAP server"}
error updating LDAP groups: {'desc': "Can't contact LDAP server"}
```

Note that I am not using LDAP. I just commented it all out in the config, and removed it from the list of authentication providers.

Deleted polls have not been deleted.

**Comment by Akasch** [comment]
The Ldap problem has been fixed in the master.

The delete is until now a delete of the url and a deleted flag in the Database, because we had problems with false deleted polls. Maybe we should delete it from we database after some (configurable) time. Note: deleting from the database will decrement the Statistics on the frontage

**Comment by RalfJung** [comment]
> The Ldap problem has been fixed in the master.

Which commit? Same as the refactoring to a local call? Can this be safely cherry-picked?

> The delete is until now a delete of the url and a deleted flag in the Database, because we had problems with false deleted polls. Maybe we should delete it from we database after some (configurable) time. Note: deleting from the database will decrement the Statistics on the frontage

Oh. I thought the delete would help to avoid an ever-growing database^^. In any case, the URLs currently remain valid, so not even that flag seems to work.

**Comment by Akasch** [comment]
I will look into it tomorrow morning.

The growing database is not such a Problem, we have ~6 MB plain sql dump with ~1.5k Polls and ~6.5k Votes

**Comment by Akasch** [comment]
> Which commit? Same as the refactoring to a local call? Can this be safely cherry-picked?

It should be save to cherry-pick the commit. With the patch you run the cronjob with:

    ./manage.py cron

**Comment by RalfJung** [comment]
Unfortunately, cherry-picking fails with a merge conflict. I will hope for version 2.0.1 then ;-)

Looking at the cronjob code, however, I noticed that it prints a message for every deleted poll. cronjobs should, IMHO, be silent when everything is all right - after all, every output will trigger an email. So it shouldn't print success, just failure.

**Comment by opatut** [comment]
The cronjob command and the verbosity fix are now part of release-2.0.1.
